scripts/xtensa-build-zephyr.py

```diff
@@ -347,6 +358,9 @@ def get_sof_version(abs_build_dir):
 		      versions['SOF_MICRO']
 	return sof_version
 
+def validate_key_name(key_name, search=re.compile(r'^[A-Za-z0-9]+$').match):
```

Comment:
This character restriction comes from me as I don't want key name with special characters. Since we're going to use `--key-type`, this is no longer needed.